cluster/gce/win1803/k8s-node-setup.psm1

```diff
@@ -318,14 +318,13 @@ function DownloadAndInstall-KubernetesBinaries {
   $tmp_dir = 'C:\k8s_tmp'
   New-Item -Force -ItemType 'directory' $tmp_dir | Out-Null
 
-  $uri = ${kube_env}['NODE_BINARY_TAR_URL']
-  $filename = Split-Path -leaf $uri
-
-  # Disable progress bar to increase download speed.
-  $ProgressPreference = 'SilentlyContinue'
-  Invoke-WebRequest $uri -OutFile ${tmp_dir}\${filename}
-
-  # TODO(yujuhong): Verify hash of the tarball.
+  $urls = ${kube_env}['NODE_BINARY_TAR_URL'].Split(",")
+  $filename = Split-Path -leaf $urls[0]
+  $hash = $null
+  if ($kube_env.ContainsKey('NODE_BINARY_TAR_HASH')) {
+    $hash = ${kube_env}['NODE_BINARY_TAR_HASH']
+  }
+  MustDownload-File -Hash $hash -OutFile ${tmp_dir}\${filename} -URLs $urls
```

Comment:
Should we change other Invoke-WebRequest calls in this file to use MustDownload-File?

Reply:
Good point. Done.